### ISSUE TITLE
Converting sklearn pipeline to pmml takes 3 days

### ISSUE THREAD

**Issue by chloris** [jpmml/sklearn2pmml#88]
Hi,
I have a PMMLPipeline model trained with 450k short sentences and the total number of classes are 47. The generated pickle model size is around 110MB. However, converting the pipeline to pmml took around 3 days. The generated pmml file size is around 1.1GB. I tried to increase the heap size (-Xms4G -Xmx16G, and I have 128GB main memory) but had no difference. Is it normal to take so much time to convert the pipeline?
My PMML pipeline is attached below. Thanks!

    pipeline = PMMLPipeline([
        ("vect", TfidfVectorizer(stop_words = 'english', norm = None, tokenizer = Splitter(), ngram_range = (1, 2), max_df = 0.7, min_df = 4)),
        ("normalizer", Normalization.Normalizer()),
        ("selector", SelectorProxy(SelectKBest(selector, k=10000))),
        ("clf", LogisticRegression(solver='sag', max_iter=500)),
    ])

**Comment by vruusmann** [comment]
Root cause: https://github.com/jpmml/jpmml-model/issues/13, other manifestation: https://github.com/jpmml/jpmml-lightgbm/issues/2

The problematic step is this: `("selector", SelectorProxy(SelectKBest(selector, k=10000)))`. Specifically, there are too many (10'000) features surviving after feature selection, which is too much for my naive cross-field dependency checker/resolver.

For example, if you bring the number of surviving features down considerably (say, `k = 500` or `k = 1000`), then the conversion should complete in mere seconds.
